TwitchPlaysAssembly/Src/ComponentSolvers/Modded/Misc/PongComponentSolver.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections;
+using UnityEngine;
+
+public class PongComponentSolver : ComponentSolver
+{
+	public PongComponentSolver(TwitchModule module) :
+		base(module)
+	{
+		component = Module.BombComponent.GetComponentInChildren(componentType);
+		selectables = Module.BombComponent.GetComponent<KMSelectable>().Children;
+		ModInfo = ComponentSolverFactory.GetModuleInfo(GetModuleType(), "!{0} left/l/right/r <top/t/middle/m/bottom/b> [Sets the left or right paddle to the specified position on the screen] | On Twitch Plays the ball travels at half its normal speed");
+		Module.StartCoroutine(TPSpeedModifier());
+	}
+
+	protected internal override IEnumerator RespondToCommandInternal(string inputCommand)
+	{
+		string[] split = inputCommand.ToLowerInvariant().SplitFull(' ');
+		if (split.Length != 2 || !split[0].EqualsAny("left", "l", "right", "r")) yield break;
+		if (!split[1].EqualsAny("top", "t", "middle", "m", "bottom", "b")) yield break;
+
+		yield return null;
+		if (split[0].EqualsAny("left", "l") && split[1].EqualsAny("top", "t"))
```

Comment:
FirstOrWhole could be used:
```suggestion
		if (split[0].FirstOrWhole("left") && split[1].FirstOrWhole("top"))
```